bucket/oh-my-posh3.json

```diff
@@ -0,0 +1,37 @@
+{
+    "version": "3.43.1",
+    "description": "A prompt theme engine for any shell",
+    "homepage": "https://ohmyposh.dev",
+    "license": {
+        "identifier": "GPL-3.0-only",
+        "url": "https://github.com/JanDeDobbeleer/oh-my-posh3/blob/main/COPYING"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/JanDeDobbeleer/oh-my-posh3/releases/download/v3.43.1/posh-windows-wsl-amd64.7z",
+            "hash": "1AEFCC4BEC99A840B87491E2ABEC37F9CD743A077331810E7BFBA86DD6E096C5"
```

Comment:
```suggestion
            "hash": "1aefcc4bec99a840b87491e2abec37f9cd743a077331810e7bfba86dd6e096c5"
```

Reply:
this hash is generate by [`$zipHash = Get-FileHash ./posh-windows-wsl-amd64.7z -Algorithm SHA256`](https://github.com/JanDeDobbeleer/oh-my-posh3/blob/2a54724a7514a8d4fcbeed2f055df5e8b11fad3b/packages/scoop/build.ps1#L22) and get uppercase result.
Next version by autoupdate will be uppercase again.

Reply:
Stop deleting comments...

You have no idea what you are talking about. The hash is extracted in autoupdate, which extract the hash from the specified and then execute `format_hash`, which return `$hash.ToLowe()`

Reply:
Sorry, I realized that I understood what you meant after performing the autoupdate, so I deleted the comment(sorry).

Reply:
I find that i use a 'checkver.ps1' from https://github.com/Ash258/GenericBucket that without feature about `$hash.ToLower()`.
Can you update scripts from [GenericBucket](https://github.com/Ash258/GenericBucket) to sync with this repo. @Ash258

Reply:
It is coming from Core not bucket specific scripts.

Reply:
https://github.com/lukesampson/scoop/blob/643cfb0da8369dbbe2964511698763c5ff179492/lib/autoupdate.ps1#L89
https://github.com/lukesampson/scoop/blob/643cfb0da8369dbbe2964511698763c5ff179492/lib/core.ps1#L856-L867

Reply:
I will not sync anything as they are exactly functinaly equal scripts...

Reply:
okay, thx.